arm_id: so101
robot_name: SO101
joint_names:
- shoulder_pan
- shoulder_lift
- elbow_flex
- wrist_flex
- wrist_roll
- gripper
target_joint_positions:
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
bounding_box:
  min_translation:
  - 1.9143362045288086
  - -0.4635680019855499
  - 0.9534375071525574
  max_translation:
  - 2.4698359966278076
  - -0.24516192078590393
  - 1.1622734069824219
  min_rotation:
  - -0.3121587038040161
  - -0.03488326072692871
  - -0.7964982390403748
  - -0.6787470579147339
  max_rotation:
  - 0.9994491338729858
  - 0.7705460786819458
  - 0.4834570586681366
  - 0.9995351433753967
cameras:
  wrist_camera:
    topic: /so101/wrist_camera/rgb
    frame_id: sim_camera
    width: 640
    height: 480
  front_camera:
    topic: /so101/front_camera/rgb
    frame_id: sim_camera
    width: 640
    height: 480
gripper:
  max_width: 0.08
  min_width: 0.0
safety:
  enable_collision_detection: true
  max_joint_velocity: 2.0
  max_joint_acceleration: 5.0
  emergency_stop_enabled: true
